UKHO.D365CallbackDistributorStub.API/UKHO.D365CallbackDistributorStub.API/Services/CallbackService.cs

```diff
@@ -42,5 +54,56 @@ public static bool SaveCallbackRequest(CallbackRequest callbackRequest, string s
             }
 
         }
+
+        public bool SaveCommandCallbackRequest(string subscriptionId, HttpStatusCode? httpStatusCode)
+        {
+            try
+            {
+                CommandCallbackRequest? commandCallbackRequest = s_commandCallbackRequestList.FirstOrDefault(a => a.SubscriptionId == subscriptionId);
```

Comment:
I think we should always consider latest matching record?

Reply:
Changes done